build.sbt

```diff
@@ -1,7 +1,18 @@
 import Dependencies._
 import Settings._
 
-scalaVersion in ThisBuild := "2.13.2"
+ThisBuild / crossScalaVersions := Seq("2.13.2", "3.0.0-M2")
```

Comment:
There is already M3 available :)

Reply:
Also, can we move to 2.13.4? Or it would require additional work?

Reply:
Yeah but there's no Fs2 dependency for `3.0.0-M3` yet. The current branch does not compile with Scala 2.13.4 and I don't want do dig down the rabbit hole right now...

Reply:
That's right, it's not worth it right now and should be done separately